jenkins/dockerized-e2e-runner.sh

```diff
@@ -73,22 +73,8 @@ fi
 # which will happen during a timeout.
 CONTAINER_NAME="${JOB_NAME}-${BUILD_NUMBER}"
 
-kill-leaked-container() {
-  local container="$1"
-  # docker runs containers with session id being the same as the pid
-  local pid=$(docker inspect --format '{{.State.Pid}}' "${container}")
-  echo "Processes running under container ${container}:"
-  ps wwuf --forest --sid "${pid}"
-  docker stop "${container}" || true
-  # Make sure artifacts are readable, since we may not have run that code
-  # in e2e-runner.sh if we were killed by timeout.
-  sudo chmod a+rX -R "${WORKSPACE}/_artifacts/" || true
-}
-
-trap "kill-leaked-container ${CONTAINER_NAME}" EXIT
-
 echo "Starting..."
-docker run --rm \
+timeout -s KILL ${DOCKER_TIMEOUT:-615m} docker run --rm \
```

Comment:
https://github.com/kubernetes/test-infra/pull/1280 assume it's going to be 700+15?